README.md

```diff
@@ -419,3 +419,22 @@ In addition to the ecosystem docs, each package has it's own README.md and `docs
 - fxa-profile-server [README](./packages/fxa-profile-server/README.md) / [docs/](./packages/fxa-profile-server/docs)
 - fxa-shared [README](./packages/fxa-shared/README.md)
 - fxa-support-panel [README](./packages/fxa-support-panel/README.md)
+
+---
+
+### Documentation for Scripts
+
+#### \_scripts/legal-md-to-pdf.sh
+
+##### Purpose
+
+This bash script allows for the conversion of markdown files to pdfs. The purpose is to create a way for legal documents to be easily converted and made available for end-users.
+
+##### Usage
+
+The script requires that you have pandoc and LaTex installed on your machine.
```

Comment:
Might help to add some example instructions on installing these things - say `apt-get install` commands or whatever.  Maybe links to the packages. Not sure who over time will need to run this script, every hint can help. Eventually, we might want to roll these into the fxa-circleci Docker container if we want to automate.

Reply:
I've updated the docs to include links to the installation guides